content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -267,6 +267,38 @@ after successful sandbox creation and network configuration by the runtime
 plugin). For a Pod without init containers, the kubelet sets the `Initialized`
 condition to `True` before sandbox creation and network configuration starts.
 
+### Pod scheduling readiness {#pod-scheduling-readiness-gate}
```

Comment:
We have a page `/docs/concepts/scheduling-eviction/pod-scheduling-readiness` dedicated to this topic.
I think we should add a link to that page instead.

Reply:
Can you elaborate where do I add the link? Simply add it in this section with a link or elsewhere in this page?

Reply:
updated.